README.md

```diff
@@ -12,25 +12,29 @@ As mentioned in the [FAQ](https://www.infracost.io/docs/faq), you can run Infrac
 
 Infracost should be run using the [Terraform directory method](https://www.infracost.io/docs/#1-terraform-directory) with this GitHub Action. Once [this issue](https://github.com/infracost/infracost/issues/394) is released, we'll be able to support other methods.
 
-### `terraform_dir`
+### `path`
 
-**Optional** Path to the Terraform code directory (default is current working directory).
+**Optional** Path to the Terraform directory or JSON/plan file. Either `path` or `config_file` is required.
 
 ### `terraform_plan_flags`
 
-**Optional** Flags to pass to the 'terraform plan' command, e.g. `"-var-file=myvars.tfvars -var-file=othervars.tfvars"`.
+**Optional** Flags to pass to the 'terraform plan' command, e.g. `"-var-file=myvars.tfvars -var-file=othervars.tfvars"`. Applicable when path is a Terraform directory.
 
 ### `usage_file`
 
-**Optional** Path to Infracost [usage file](https://www.infracost.io/docs/usage_based_resources#infracost-usage-file) that specifies values for usage-based resources, see [this example file](https://github.com/infracost/infracost/blob/master/infracost-usage-example.yml) for the available options. The file should be present in the master/main branch too.
+**Optional** Path to Infracost [usage file](https://www.infracost.io/docs/usage_based_resources#infracost-usage-file) that specifies values for usage-based resources, see [this example file](https://github.com/infracost/infracost/blob/master/infracost-usage-example.yml) for the available options.
 
-### `percentage_threshold`
+### `usage_file`
 
-**Optional** The absolute percentage threshold that triggers a pull request comment with the diff. Defaults to 0, meaning that a comment is posted if the cost estimate changes. For example, set to 5 to post a comment if the cost estimate changes by plus or minus 5%.
+**Optional** Path to Infracost [usage file](https://www.infracost.io/docs/usage_based_resources#infracost-usage-file) that specifies values for usage-based resources, see [this example file](https://github.com/infracost/infracost/blob/master/infracost-usage-example.yml) for the available options.
+
+### `config_file`
 
-### `pricing_api_endpoint`
+**Optional** Path to the Infracost config file. Cannot be used with path, terraform* or usage-file flags.
 
-**Optional** Specify an alternate Cloud Pricing API URL (default is https://pricing.api.infracost.io).
+### `percentage_threshold`
+
+**Optional** The absolute percentage threshold that triggers a pull request comment with the diff. Defaults to 0, meaning that a comment is posted if the cost estimate changes. For example, set to 5 to post a comment if the cost estimate changes by plus or minus 5%.
```

Comment:
hmm we say a comment is posted if the threshold is hit, but the bash script says `Comment not posted as diff ($absolute_percent%) is less than or equal to percentage threshold`. I think I prefer the former as it works with the default 0 threshold too.